news/_posts/core-weekly/2021-02-01-coreweekly-04-2021.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 4 of 2021"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2021-02-01
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+twitter_image: /assets/images/theme/banner-core-weekly.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 25th to Sunday 31th of January 2021.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## Releases
+
+* [slint-config](https://github.com/PrestaShop/eslint-config): [v0.1.0](https://github.com/PrestaShop/eslint-config/releases/tag/0.1.0)
```

Comment:
does this need an alias, as it has been done for modules and other tools?

Reply:
Just a mistake. I wanted to remove the lowercase e and replace it with an uppercase E but I did only half of the work. Fixed.